treeherder/model/models.py

```diff
@@ -1251,6 +1251,23 @@ def elastic_search_insert(self):
             return es_line
 
 
+class Group(models.Model):
+    """
+    The test harness Group.  Sometimes this is a Manifest, but not always.
+    Not to be confused with JobGroup which is Treeherder specific.
+    """
+    id = models.BigAutoField(primary_key=True)
+    name = models.CharField(max_length=200, unique=True)
```

Comment:
Also, I did find some paths that were absolute (all xpcshell that I can find so far) so I filed a bug to @ahal for that: https://bugzilla.mozilla.org/show_bug.cgi?id=1373796